pkg/controller/deployment/deployment_overrides_test.go

```diff
@@ -92,9 +92,7 @@ func TestUnsupportedConfigOverrides(t *testing.T) {
 					Args: testArgsToAppend,
 				},
 			},
-			wantArgs: []string{
-				"--v=2", "--cluster-resource-namespace=$(POD_NAMESPACE)", "--leader-election-namespace=kube-system",
-			},
+			wantArgs: deploymentDefaultArgs["cert-manager"],
```

Comment:
Tests shouldn't be re-using the code that it testing. So I would prefer to have the explicit strings here instead of using the variable that is contains values from the manifest file (we want to test what is in the manifest file).

Reply:
I would prefer if this was the case in the other tests too, but lets not actively revert this.

Reply:
Thanks, I've made these changes in: 3f0b8ea0a2ca370d4121c9b54b2d0a16aa3f7bb8 to ensure the said behaviour is followed.